api/src/crons/index.js

```diff
@@ -0,0 +1,50 @@
+const cron = require("node-cron");
+const YoungModel = require("../models/young");
+const { capture } = require("../sentry");
+
+// dev : */5 * * * * * (every 5 secs)
+
+// every day at 4am
+cron.schedule("0 4 * * *", function () {
+  console.log("START CRON: auto validation");
+  autoValidate();
+});
+
+const autoValidate = async () => {
+  let count = 0;
+
+  const cursor = await YoungModel.find({
+    status: "WAITING_VALIDATION",
+    cohort: "2021",
+  }).cursor();
+
+  await cursor.eachAsync(async (doc) => {
+    try {
+      console.log("checking", doc.email);
+
+      // check the last item in historic with status WAITING_VALIDATION
+      const lastStatus = getLastStatus(doc, "WAITING_VALIDATION");
+
+      // if its createdAt > 2w, then validated by cron
+      if (gt2w(lastStatus.createdAt)) {
+        count++;
```

Comment:
(optionnel) Pour savoir qui est concerné (et du coup la partie `console.log("checking", doc.email);` devient pas importante et supprimable).

```suggestion
        console.log("updating", doc.email);
        count++;
```